mobile/src/home-view.js

```diff
@@ -107,8 +107,9 @@ class HomeView extends Component {
 
   organizeFilters = () => {
     var filters = this.state.filters
+    var questions = Object.values(this.state.questions).filter(question => question.block === false)
     if (filters.length === 0) {
-      Object.values(this.state.questions).map((item) => {
+      questions.map((item) => {
```

Comment:
Use `forEach` instead of `map` since you are not mapping the array *to* anything but instead *doing* something for each item.